tanzu-application-platform/README.md

```diff
@@ -0,0 +1,178 @@
+DNS registrar and create an appropriate DNS entry for the hostname of the application.  This documentation assumes you will use the hostname `animal-rescue.  The full URL 
+in this case would be:
+
+```
+animal-rescue.<your domain name>
+```
+
+Make a note of this URL as you will need it through various steps of the following doc.
+
+It is assumed that you have cloned the Animal Rescue Git repository to your workstation.  All of the `ytt` commands below should be run from the
+`tanzu-application-platform` directory in the cloned repository.
+
+### Testing Pipeline Deployment
+
+If you have a supply chain configured that includes testing, you will need to install the testing pipeline used when a build is performed.  Run the following command 
+to install the pipeline replacing the following placeholder:
+
+- **<workloadNamespace>** – Namespace where the application will be deployed
+
+```
+kubectl apply -f testingPipeline.yml -n <workloadNamespace>
+```
+
+For example:
+
+```
+kubectl apply -f testingPipeline.yml -n workloads
+```
+
+### Let's Encrypt CA Issuer
+
+Animal Rescue enables TLS connections for security and reliable transport and utilizes the Let's Encrypt production CA. Install the Let's Encrypt
+CA issuer into your cluster by running the following command and replacing the following place holder:
+
+- **<email>** – The email address of the subject responsible for the issued certificates.
+ 
+```
+ytt -f caIssuer.yml -v email=<email> | kubectl apply -f-
+```
+
+For example:
+
+```
+ytt -f caIssuer.yml -v email=joe@joesgarage.com | kubectl apply -f-
+```
+
+### AppSSO Deployment
+
+Animal Rescue requires the use of an AppSSO authorization server and client registration resource. 
+
+Deploy the authorization server instance by running the following commands and replacing these place holders:
+
+- **<serviceNamespace>** – Namespace where service instances will be deployed
+- **<devDefaultAccountUsername>** – Username for authentication
+- **<devDefaultAccountPassword>** – Password for authentication
+
+```
+ytt -f appSSOInstance.yml -v serviceNamespace=<serviceNamespace> -v devDefaultAccountUsername=<devDefaultAccountUsername> -v devDefaultAccountPassword=<devDefaultAccountPassword> | kubectl apply -f-
+```
+
+For example:
+
+```
+ytt -f appSSOInstance.yml -v serviceNamespace=service-instances -v devDefaultAccountUsername=animal -v devDefaultAccountPassword=rescue | kubectl apply -f-
+```
+
+Next, create a ClassClaim resource by running the following command and replacing the `<workloadNamespace>` placeholder with the namespace where the application will be deployed. 
+This will also created the secret that will contain the AppSSO credential information that will used by the Spring Cloud Gateway.
+
+- **<workloadNamespace>** – Namespace where the application will be deployed
+
+```
+ytt -f workloadRegistrationResource.yml -v workloadNamespace=<workloadNamespace>| kubectl apply –f-
```

Comment:
I think the character `–f-` is not using the correct dash, I had to use the command below to get it to run:

```suggestion
ytt -f workloadRegistrationResource.yml -v workloadNamespace=<workloadNamespace> | kubectl apply -f-
```

Reply:
Resolved in the PR.  Nice catch.